### ISSUE TITLE
Remove hack in Ray initialization to support out of core

### ISSUE THREAD

**Issue by devin-petersohn** [modin-project/modin#823]
### Describe the problem

We introduced a hack in #822 to be able to support out of core and larger memory hungry dataframes.

This is blocked on ray-project/ray#5837

### Source code / logs

https://github.com/modin-project/modin/pull/822/files#diff-8336a42cad8bd69c512c0a368d8aa710R136-R138

**Comment by devin-petersohn** [comment]
No longer relevant as we downgraded our Ray version.
